src/components/SectionHeader/index.js

```diff
@@ -0,0 +1,2 @@
+import SectionHeader from './SectionHeader';
```

Comment:
Consider shortening/changing to
`export { default } from 'components/SectionHeader';`